app/views/users/edit.html.haml

```diff
@@ -31,7 +31,11 @@
   = label_tag(:time, 'Time available to cook per day')
   = number_field_tag(:time, 'time', min: 1, required: true)
 
-  = label_tag(:cuisine, 'Cuisine Preferences')
-  = text_field(:cuisine, 'cuisine')
+  = label_tag(:dietary_preferences, 'Dietary Preferences')
+  = check_box(:dietary_preferences, "dietary_preferences", {:multiple => true}, "vegetarian", nil)
```

Comment:
This is cool! I thought we wouldn't be able to do dietary preferences, I'm glad that we were able to add the ability to filter by things like vegetarianism.